### DISCUSSION TITLE
sigma-file-manager-git (2.0.0.alpha.1.r39.gf0768f5-1)

### DISCUSSION THREAD

**Discussion by RainerMathey** [aleksey-hoffman/sigma-file-manager#350] [Q&A]
... hi, i've installed sigma-file-manager-git (2.0.0.alpha.1.r39.gf0768f5-1) on EndeavourOS (Archlinux) is there an known error, because when i start it, it says v1.7.0 ?
![sigma](https://github.com/user-attachments/assets/23258cf3-5f1d-4193-8c65-9fe9217b91ef)

**Comment by aleksey-hoffman** [comment]
Hi, the v2.0 doesn't have this page yet. I think it actually installed v1.7. 
I didn't upload it to Arch repo, so I'm not sure what peple put there

**Reply by RainerMathey** [reply]
... when you will upload it to the Arch repo ?
